### ISSUE TITLE
[Bug]: Stuck at 0% downloading

### ISSUE THREAD

**Issue by Soliux** [YTVanced/VancedManager#734]
### Device

Device: Samsung galaxy s9
Operating System: Android 
Android Version: 10.0

### Version

Latest

### Bug Description

I rooted my s9 and then I went to download the vanced manager and try to dow load the youtubw vanced from there, it just stays at 0% and does not move forward. I left it for 30 minutes and still nothing.

### Additional checks

- [X] I have checked other bug reports and this is not a duplicate.
- [X] This is a bug in Vanced Manager and NOT YouTube Vanced/YouTube Vanced Music/Vanced microG.

**Comment by Soliux** [comment]
![Screenshot_20211106-162327_Vanced Manager](https://user-images.githubusercontent.com/66280234/140616722-26fd7bab-bef2-4b92-b9d5-df97469c0f2c.jpg)

**Comment by RUvlad1** [comment]
Try to download with a VPN on?

**Comment by TeaPack97** [comment]
Same error on Xiaomi redmi 6 with MIUI 11.0. 4.
At this mobile i installed it 2 years ago but latelly i am unable to update anything.
After clean install unable to download microG

**Comment by dmt0** [comment]
Same on rooted Samsung S10e. Found a workaround on Reddit:
Had to go into Settings > Apps > Vanced
Manually grant permission for file system access
Allow to install apps from source "Vanced"

Basically Vanced Manager is not requesting permissions properly. That's all.

But then, after all the apks finished downloading, I'm stuck at "Please do NOT exit the app during this process!" and the installation never seems to start.

**Comment by Soliux** [comment]
Yeah I had quite a few issues getting the rooted version of YouTube Vanced installed. I just go the non rooted version to install instead but I did have to switch over to mobile data for me to be able to download it.
